Formula/octave.rb

```diff
@@ -54,6 +54,8 @@ class Octave < Formula
   depends_on "texinfo"
   depends_on "veclibfort"
 
+  depends_on "qt" => :recommended
```

Comment:
From our policy (https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#stuff-that-builds-a-gui-by-default-but-doesnt-have-to):

> **Stuff that builds a GUI by default (but doesn't have to)**
> Make it build a command-line tool or a library by default and, if the GUI is useful and would be widely used, add an option to build the GUI. Don't offer an option for multiple GUI backends e.g. X11 is a bad user experience for GUIs on macOS.

So if we want to add qt, it should be `:optional`, not `:recommended`. Octave has many users who simply use the command line.

Reply:
Okay, fixed.